### PR TITLE
Add keycload vars as defaults

### PR DESCRIPTION
**Pull request by danypr92**

### CODE REVIEW

File: inventory/host_vars/stagingtryton.coopdevs.org/keycloak.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+keycloak_client_secret: !vault |
+          $ANSIBLE_VAULT;1.1;AES256
+          66393934626266313533393234393662633666333539643064633831306433393736643632363562
+          3162346433343562653265333533323036613835366663320a363331646136383535613063656666
+          36653834323736373161386566663933393737616666653138643930393261303066316130353862
+          3863623566636437610a383731356332616530346332333664643663636132613432666464303834
+          66626263306235313961373065303364356432333432613761316363353032323562653432613235
+          3264643134646434616632376231663863643566396230663464
+keycloak_real: "opencell"
```
Comment (enricostano) [inline inventory/host_vars/stagingtryton.coopdevs.org/keycloak.yml:20]: realm?

File: roles/systemd_service/templates/defaults_trytond.j2 [hunk 2]
```diff
@@ -44,3 +44,9 @@ SOMOFFICE_URL="{{ somoffice_url }}"
 SOMOFFICE_USER="{{ somoffice_user }}"
 SOMOFFICE_PASSWORD="{{ somoffice_password }}"
 SOMOFFICE_USER_CREATION_ENABLED="{{ somoffice_user_creation_enabled }}"
+
+KEYCLOAK_AUTHENTICATION_ENABLED="{{ keycloak_authentication_enabled }}"
+KEYCLOAK_SERVER_URL="{{ keycloak_url }}"
+KEYCLOAK_CLIENT_ID="{{ keycloak_client_id }}"
+KEYCLOAK_CLIENT_SECRET="{{ keycloak_client_secret }}"
+KEYCLOAK_REALM="{{ keycloak_real }}"
```
Comment (enricostano) [inline roles/systemd_service/templates/defaults_trytond.j2:52]: realm?

File: inventory/host_vars/stagingtryton.coopdevs.org/keycloak.yml (hunk 1 above)
Comment (danypr92) [reply inventory/host_vars/stagingtryton.coopdevs.org/keycloak.yml]: Fixed

File: roles/systemd_service/templates/defaults_trytond.j2 (hunk 2 above)
Comment (danypr92) [reply roles/systemd_service/templates/defaults_trytond.j2]: Fixed

**Pull request merged**